### DISCUSSION TITLE
Al que le falte equipo para hackaton (hasta 5 integrantes)!

### DISCUSSION THREAD

**Discussion by Alejandra-Adalid** [Launch-X-Latam/MisionFrontEnd#551] [Team Posts]
Pasen emails de innovacion y los agrego al grupo !

**Comment by fabiolaodioInnova** [comment]
Hola Alejandra, mi correo es ***@***.***

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Alejandra-Adalid ***@***.***>
Sent: Friday, November 11, 2022 2:30:41 PM
Cc: Subscribed ***@***.***>
Subject: [Launch-X-Latam/mision-backend] Al que le falte equipo para hackaton (hasta 5 integrantes)! (#87)


Pasen emails de innovacion y los agrego al grupo que cree !

**Comment by edison-sc** [comment]
Hola Ale, mi correo es lx01980@innovaccion.mx

**Comment by Alejandra-Adalid** [comment]
> lx01980@innovaccion.mx

ya te agregue Edison. !!

**Comment by Alejandra-Adalid** [comment]
Faltan 3

**Comment by OmarGarcia20** [comment]
Hola, mi correo es lx01934@innovaccion.mx

**Comment by Alejandra-Adalid** [comment]
> lx01934@innovaccion.mx

ya estas agregado, Omar !

**Comment by Alejandra-Adalid** [comment]
faltan 2 mas !

**Comment by itsgaby** [comment]
Hola alejandra, mi correo es lx05797@innovaccion.mx

**Comment by OmarGarcia20** [comment]
> lx01934@innovaccion.mx
> 
> ya estas agregado, Omar !

Gracias!!!

**Comment by MarcoAntonio159** [comment]
Hola, si aun tienen especio, estoy dispoenible 
lx05663@innovaccion.mx

**Comment by Alejandra-Adalid** [comment]
> lx05663@innovaccion.mx

ya te agregue, Iner

**Comment by Alejandra-Adalid** [comment]
FALTA UNO !

**Comment by LaunchxUser** [comment]
Comparto mi correo
***@***.***

gracias

________________________________
De: Alejandra-Adalid ***@***.***>
Enviado: viernes, 11 de noviembre de 2022 02:40 p. m.
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Al que le falte equipo para hackaton (hasta 5 integrantes)! (#87)


Faltan 3

**Comment by MarcoAntonio159** [comment]
> lx05663@innovaccion.mx
> 
> ya te agregue, Iner

Gracias, como puedo ver que estoy en tu grupo?

**Comment by IsraelVera** [comment]
Mi correo es lx04916@innovaccion.mx mi nombre es Israel

**Comment by Tintto** [comment]
Hola .. soy Adriana no tengo grupo

**Comment by fabiolaodioInnova** [comment]
Ya tienen completo el grupo?
________________________________
De: Alejandra-Adalid ***@***.***>
Enviado: viernes, 11 de noviembre de 2022 14:48
Cc: Fabiola de los Angeles Odio ***@***.***>; Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Al que le falte equipo para hackaton (hasta 5 integrantes)! (#87)


Compañeros! yaa cree el grupo, fijense que le llego una solicitud por teams ! si no el link es

https://teams.microsoft.com/l/team/19%3aWX-eqvd3xljsjLznnQ5dIdTUyMV4dr9nPUAGns3AlnA1%40thread.tacv2/conversations?groupId=8101ccf0-bee4-44ed-83c0-61874367d573&tenantId=4ae54b05-b77e-4224-aef1-8661422e0816

**Comment by Alejandra-Adalid** [comment]
ya estan esta completo..

**Comment by Alejandra-Adalid** [comment]
Bueno se cerro somos 5:
 lx01980 lx01934 lx06222 lx04916 lx05663

**Comment by R3D4N** [comment]
aun tienes espacio para otro integrante? porsi mi correo es lx10972@innovaccion.mx

**Comment by fabiolaodioInnova** [comment]
Los agregué a mi grupo ***@***.******@***.***> y ***@***.******@***.***> , nos faltarían dos integrantes.
________________________________
De: R3_D4N ***@***.***>
Enviado: viernes, 11 de noviembre de 2022 14:58
Cc: Fabiola de los Angeles Odio ***@***.***>; Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Al que le falte equipo para hackaton (hasta 5 integrantes)! (#87)


aun tienes espacio para otro integrante? porsi mi correo es ***@***.******@***.***>

**Comment by DanielSkillDev** [comment]
Yo me estoy poniendo al corriente pero estoy disponible 
lx03824@innovaccion.mx

**Comment by cortazarluis** [comment]
Tambien estoy disponible lx16369@innovaccion.mx

**Comment by fabiolaodioInnova** [comment]
Listo, ya los agregué. Estamos completos
________________________________
De: cortazarluis ***@***.***>
Enviado: viernes, 11 de noviembre de 2022 15:14
Cc: Fabiola de los Angeles Odio ***@***.***>; Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Al que le falte equipo para hackaton (hasta 5 integrantes)! (#87)


Tambien estoy disponible ***@***.******@***.***>

**Comment by cortazarluis** [comment]
Gracias

**Comment by kerizrh** [comment]
Hola, a mi me falta grupo. lx07570@innovaccion.mx

**Comment by Geomarti** [comment]
Este es mi correo ***@***.*** <http://innovaccion.mx/>*

**Comment by JGSergio** [comment]
saludos alejandra no tengo grupo este es mi correo lx08533@innovaccion.mx

**Comment by maurpz** [comment]
agregenme si les falta 1 mas porfa  lx05608@innovaccion.mx

**Comment by RuruDuran** [comment]
Hola si queda un espacio me pueden agregar? Soy diseñador gráfico lx14115@innovaccion.mx

**Comment by AbnerHub** [comment]
Hola me podrian agregar mi correo es lx02522@innovaccion.mx

**Comment by innovategm** [comment]
aun les falta integrante? 
lx05800@innovaccion.mx
